app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -32,6 +35,8 @@ class RecentPatientsViewController @Inject constructor(
     @Named("full_date") private val dateFormatter: DateTimeFormatter
 ) : ObservableTransformer<UiEvent, UiChange> {
 
+  private val today = Instant.now(userClock).toLocalDateAtZone(userClock.zone)
```

Comment:
This need not be a class property, right? We could instantiate it inside the operator where it is being used.

```kotlin
.map { 
  val today = LocalDate.now(userClock)
  it.map { recentPatient -> recentPatientItem(recentPatient, today) } 
}
```